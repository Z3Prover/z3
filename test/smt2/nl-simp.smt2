
; Copyright (c) 2015 Microsoft Corporation

(simplify (not (= (root-obj (+ (* 64 (^ x 2)) (- 65)) 2) 0.0)))

(simplify (= (root-obj (+ (* 64 (^ x 2)) (- 65)) 2) 0.0))

(check-sat)

(eval (not (= (root-obj (+ (* 64 (^ x 2)) (- 65)) 2) 0.0)))

(eval (= (root-obj (+ (* 64 (^ x 2)) (- 65)) 2) 0.0))
