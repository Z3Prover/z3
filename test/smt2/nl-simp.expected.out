true
false
sat
true
false
